abb_bringup/launch/abb_moveit.launch.py

```diff
@@ -32,94 +30,47 @@ def launch_setup(context, *args, **kwargs):
     moveit_config_package = LaunchConfiguration("moveit_config_package")
     moveit_config_file = LaunchConfiguration("moveit_config_file")
 
-    # Planning context
-    robot_description_content = Command(
-        [
-            PathJoinSubstitution([FindExecutable(name="xacro")]),
-            " ",
-            PathJoinSubstitution(
-                [FindPackageShare(support_package), "urdf", robot_xacro_file]
-            ),
-        ]
-    )
-    robot_description = {"robot_description": robot_description_content}
-
-    robot_description_semantic_content = Command(
-        [
-            PathJoinSubstitution([FindExecutable(name="xacro")]),
-            " ",
-            PathJoinSubstitution(
-                [FindPackageShare(moveit_config_package), "config", moveit_config_file]
-            ),
-        ]
+    # MoveIt configuration
+    moveit_config = (
+        MoveItConfigsBuilder("abb_irb1200_5_90")
```

Comment:
Let's not hardcode any particular moveit config package. By passing the pkg name here, the builder will retrieve relevant files from the pkg's config subfolder if present.
```suggestion
        MoveItConfigsBuilder("abb_bringup", package_name=f'{moveit_config_package.perform(context)}')
```